runtime/tr.source/trj9/optimizer/J9TransformUtil.cpp

```diff
@@ -182,6 +214,11 @@ static bool verifyFieldAccess(void *curStruct, TR::SymbolReference *field, TR::C
       TR_YesNoMaybe objectContainsField = fej9->isInstanceOf(objectClass, fieldClass, true);
       return objectContainsField == TR_yes;
       }
+   // Check for VarHandle.handleTable, it can be fabricated by VarHandleTransformer
+   else if (isFabricatedVarHandleField(field, comp))
+      {
+      return true;
```

Comment:
This should still verify the instance, because dead paths could load `VarHandle.handleTable` from a known object that isn't a `VarHandle`